langchain/src/document_loaders/fs/openai_whisper_audio.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Configuration,
+  ConfigurationParameters,
+  CreateTranscriptionResponse,
+  OpenAIApi,
+} from "openai";
+import { Document } from "../../document.js";
+import { BaseDocumentLoader } from "../base.js";
+
+const MODEL_NAME = "whisper-1";
+const MAX_FILE_SIZE = 25 * 1024 * 1024; // 25MB
```

Comment:
I would remove this and just let OpenAI throw the error? They could increase this on their end in the future.